pkgs/development/tools/conftest/default.nix

```diff
@@ -13,16 +13,24 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-jI5bX6S2C0ckiiieVlaRNEsLS/5gGkC3o/xauDtCOjA=";
 
-  doCheck = false;
-
   ldflags = [
     "-s"
     "-w"
     "-X github.com/open-policy-agent/conftest/internal/commands.version=${version}"
   ];
 
+  HOME = "$TMPDIR";
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/conftest --version | grep ${version} > /dev/null
```

Comment:
```suggestion
  

  doInstallCheck = true;
  installCheckPhase = ''
    export HOME=$(mktemp -d)
    $out/bin/conftest --version | grep ${version} > /dev/null
```

Reply:
Adding `export HOME=$(mktemp -d)` to installCheckPhase and preCheckPhase fails:

```
> === RUN   TestFind/should_return_Data_Dir_with_cache_if_XDG_DATA_DIRS_is_set.
       >     xdg_test.go:232: unexpected error stat /homeless-shelter: no such file or directory
       > --- FAIL: TestFind (0.00s)
       >     --- PASS: TestFind/should_return_error_if_dir_does_not_exist. (0.00s)
       >     --- FAIL: TestFind/should_return_homeDir_if_no_XDG_path_is_set. (0.00s)
       >     --- FAIL: TestFind/should_return_XDG_DATA_HOME_if_XDG_DATA_HOME_is_set. (0.00s)
       >     --- FAIL: TestFind/should_return_Data_Dir_with_cache_if_XDG_DATA_DIRS_is_set. (0.00s)
       > FAIL
       > FAIL	github.com/open-policy-agent/conftest/plugin	0.010s
       > FAIL
```
Any idea why it fails?

Also tried w/o success:
`export XDG_DATA_HOME=$(mktemp -d)`
`export XDG_DATA_DIRS=$(mktemp -d)`

Tried also at `preCheckPhase` and `installCheckPhase`.

So far adding `HOME = "$TMPDIR";` is what worked.

Reply:
Then it might be hardcoded at build time. You can changed the variable scope. Nothing to worry about at this point.

Reply:
Change the variable scope as in what? I'm sorry but I did not understand.
I have no change in mind. If there is one to be done, let me know. Thanks.

Reply:
You can keep it like it is now and have HOME defined everywhere. Better like this rather than that it does not work.

Reply:
Right, it is already so. I just need someone to approve+merge this. Can you? @SuperSandro2000